### ISSUE TITLE
Bad yolo cfg file

### ISSUE THREAD

**Issue by iraadit** [cvjena/darknet#5]
Hi,

The config file yolo.cfg doesn't seem to work anymore.
When using this config file, the probabilities are really high and YOLO is detecting objects everywhere.

Using yolo2.cfg (or yolo.cfg that is in pjreddie's repository), the detection is going all right.

**Comment by iraadit** [comment]
I think that as pjreddie has updated the yolo.cfg file (one [local] and one [dropout] layers instead of the before last [connected] layer), he has also updated the pre-trained weights available on his website.

Using your code with his config and weights file is working; but your config file isn't compatible anymore with the weights file available on his website.
